apps/charterafrica/src/components/Articles/Articles.js

```diff
@@ -1,17 +1,25 @@
 import { Section } from "@commons-ui/core";
 import { Box, Grid } from "@mui/material";
 import PropTypes from "prop-types";
-import { forwardRef } from "react";
-
-import PostCard from "../Postcard";
+import React from "react";
 
 import { secondary } from "@/charterafrica/colors";
+import PostCard from "@/charterafrica/components/PostCard";
 
-const Articles = forwardRef((props, ref) => {
+const Articles = React.forwardRef((props, ref) => {
   const { sx, articles } = props;
   return (
     <Box bgcolor={secondary[50]} sx={sx} ref={ref}>
-      <Section sx={{ px: { xs: "57.5px", sm: 0 } }}>
+      <Section
+        sx={{
+          px: {
+            xs: "57.5px",
+            // (theme.contentWidths.values.sm - 590) / 2
```

Comment:
```suggestion
```

Reply:
This explains how we arrive at 69px used on the next line @koechkevin . It's not given directly on the design. 2 months from now I'll certainly not remember.